lms/templates/video.html

```diff
@@ -59,7 +60,16 @@
               <div id="${id}"></div>
               <h3 class="hidden">${_('No playable video sources found.')}</h3>
           </section>
+
+          <a href="#" tabindex="0" class="hide-captions" role="button" aria-disabled="false"
+                title="${_('Turn off transcripts')}">
+            <i class="icon-caret-right" aria-hidden="true"></i>
```

Comment:
We've recently updated our Font Awesome library, so you'll want to update the icons to use the [latest Font Awesome set] (http://fontawesome.io/cheatsheet/). A rebase against Master might be wise too.